bitnami/spring-cloud-dataflow/templates/server/configmap.yaml

```
@@ -83,7 +83,7 @@ data:
         driverClassName: {{ include "scdf.database.driver" . }}
         username: {{ include "scdf.database.server.user" . }}
         {{ if .Values.externalDatabase.existingPasswordSecret }}
-        password: ${datasource-password}
+        password: {{ .Values.externalDatabase.existingPasswordKey | default "datasource-password" | printf "${%s}" }}
```

This should be like:
```suggestion
        password: ${{{ .Values.externalDatabase.existingPasswordKey | default "datasource-password" | printf "${%s}" }}}
```

shouldn't it?

---

Sorry, I missed the last part `| printf "${%s}"`